sources/packages/web/src/types/contracts/Common.ts

```diff
@@ -1,12 +1,23 @@
-// TODO: When all icon are replace to font awesome,
-// replace  icon?: string | string[] with
-// icon?:string[];
+import { RouteLocationRaw } from "vue-router";
+
+// todo: use MenuInterface and once MenuModel removed from all, delete the MenuModel
 export interface MenuModel {
   label: string;
   icon?: string | string[];
   command?: () => void;
   items?: MenuModel[];
 }
+// Interface for Menu.
+export interface MenuInterface {
```

Comment:
Can we keep here only the properties currently needed for this PR and expand the object as needed?